src/components/input/_macro.njk

```diff
@@ -51,6 +51,7 @@
                     {% if inputmode is defined %}inputmode="{{ inputmode }}"{% endif %}
                     {% if params.autocomplete is defined %}autocomplete="{{ params.autocomplete }}"{% endif %}
                     {% if params.charcheck %}data-char-check-ref="{{ params.id }}-check-remaining" data-char-check-num="{{ params.charcheck }}"{% endif %}
+                    {% if params.charcheck %}data-char-check-ref="{{ params.id }}-check-remaining" data-char-check-num="{{ params.charcheck }}" {%if params.charcheckMinOnly %}data-char-check-min-only="{{ params.charcheckMinOnly }}"{% endif %}{% endif %}
```

Comment:
The condition is duplicating whats above. The new condition just needs to be added to the existing check.